website/docs/r/project.html.markdown

```diff
@@ -10,8 +10,6 @@ description: |-
 
 `mongodbatlas_project` provides a Project resource. This allows project to be created.
 
```

Comment:
@andreaangiolillo in docs while we are here can also update below section in docs to remove hard code of role names and link to our docs?

role_names - (Required) Each string in the array represents a project role you want to assign to the team. Every user associated with the team inherits these roles. You must specify an array even if you are only associating a single role with the team. The following are valid roles:

GROUP_OWNER
GROUP_READ_ONLY
GROUP_DATA_ACCESS_ADMIN
GROUP_DATA_ACCESS_READ_WRITE
GROUP_DATA_ACCESS_READ_ONLY
GROUP_CLUSTER_MANAGER

Reply:
Updated